feelpp/feel/feelmodels/modelmaterials.hpp

```diff
@@ -432,6 +432,36 @@ class FEELPP_EXPORT ModelMaterials: public std::map<std::string,ModelMaterial>,
     void setParameterValues( std::map<std::string,double> const& mp );
 
     void saveMD(std::ostream &os);
+
+    /**
+     * get the list of symbols associated to expressions
+     */
+    auto symbolsExpr( bool add_evaluable = false ) const
```

Comment:
the implementation is not correct with multi-material. The id `symbName` should depend on material name else it will be overridden or ignored in the symbolsExpr structure.

Reply:
The name is now composed like this "<material>_<property>"